### ISSUE TITLE
Too many logging when training with TensorFlow-Slim image classification model library

### ISSUE THREAD

**Issue by ohadlights** [tensorflow/models#3276]
### System information
- **What is the top-level directory of the model you are using**: slim
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**: no
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: Windows/Ubuntu
- **TensorFlow installed from (source or binary)**: binary
- **TensorFlow version (use command below)**: 1.5
- **Bazel version (if compiling from source)**: N/A
- **CUDA/cuDNN version**: 9
- **GPU model and memory**: Many...
- **Exact command to reproduce**: run script slim/train_image_classifier.py

### Describe the problem
When training using the mentioned script (train_image_classifier.py) there are log prints each time a new summary is written to the events file. 

For example:
INFO:tensorflow:Recording summary at step 16923

I think it makes the log very noisy. There should be a way to disable it or print it as debug.

**Comment by monomon** [comment]
Have you considered the option of raising your log level to warning?

**Comment by ohadlights** [comment]
Yes. but there are logs that I do want to see such as:
INFO:tensorflow:global step 36000: loss = 4.4917 (0.314 sec/step)

or:
INFO:tensorflow:Saving checkpoint to path /mnt/person_data/Training/Recognition/Models/TensorFlow/Test/jianguo_srt_multi2_2/model.ckpt

**Comment by drpngx** [comment]
You can use `tf.logging.set_verbosity` in your program to set the levels of messages that you want to see.

This question is better asked on  [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) since it is not a  bug or feature request. There is also a larger community that reads questions there. Thanks!
